src/store/entriesSlice.ts

```diff
@@ -1,20 +1,25 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createSlice } from "@reduxjs/toolkit";
 
-
-const initialState = {
-    entries: {}
+export interface entriesInitialStateType {
+  entries: [];
```

Comment:
should this be an object with the values it will take in per entry?

Reply:
Yes, we can definitely do so. For now, it is adopting the types as given from backends return.